cronos/chain.json

```diff
@@ -25,30 +21,29 @@
   },
   "codebase": {
     "git_repo": "https://github.com/crypto-org-chain/cronos",
-    "recommended_version": "v1.2.0",
-    "compatible_versions": [],
+    "recommended_version": "v1.2.1",
+    "compatible_versions": ["v1.2.0"],
     "binaries": {
-      "linux/amd64": "https://github.com/crypto-org-chain/cronos/releases/download/v1.2.0/cronos_1.2.0_Linux_x86_64.tar.gz",
-      "linux/arm64": "https://github.com/crypto-org-chain/cronos/releases/download/v1.2.0/cronos_1.2.0_Linux_arm64.tar.gz",
-      "darwin/amd64": "https://github.com/crypto-org-chain/cronos/releases/download/v1.2.0/cronos_1.2.0_Darwin_x86_64.tar.gz",
-      "darwin/arm64": "https://github.com/crypto-org-chain/cronos/releases/download/v1.2.0/cronos_1.2.0_Darwin_arm64.tar.gz",
-      "windows/amd64": "https://github.com/crypto-org-chain/cronos/releases/download/v1.2.0/cronos_1.2.0_Windows_x86_64.zip"
+      "linux/amd64": "https://github.com/crypto-org-chain/cronos/releases/download/v1.2.1/cronos_1.2.1_Linux_x86_64.tar.gz",
+      "linux/arm64": "https://github.com/crypto-org-chain/cronos/releases/download/v1.2.1/cronos_1.2.1_Linux_arm64.tar.gz",
+      "darwin/amd64": "https://github.com/crypto-org-chain/cronos/releases/download/v1.2.1/cronos_1.2.1_Darwin_x86_64.tar.gz",
+      "darwin/arm64": "https://github.com/crypto-org-chain/cronos/releases/download/v1.2.1/cronos_1.2.1_Darwin_arm64.tar.gz",
+      "windows/amd64": "https://github.com/crypto-org-chain/cronos/releases/download/v1.2.1/cronos_1.2.1_Windows_x86_64.zip"
     },
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/crypto-org-chain/cronos-mainnet/master/cronosmainnet_25-1/genesis.json"
     },
     "versions": [
       {
-        "name": "v1.2.0",
-        "recommended_version": "v1.2.0",
-        "compatible_versions": [
-        ],
+        "name": "v1.2.1",
+        "recommended_version": "v1.2.1",
+        "compatible_versions": ["v1.2.0"],
```

Comment:
Please return the formatting of this section and key_algos and extra_codecs to original format, and add v1.2.1 to the list of compatible versions.

Reply:
Thanks @reversesigh, updated accordingly and thanks for your review!